.github/workflows/alpha-release.yml

```diff
@@ -0,0 +1,79 @@
+name: Manual Alpha Package Release
+
+on:
+  # Allows running workflow manually from the GitHub Actions tab
+  workflow_dispatch:
+
+jobs:
+  build_and_test:
+    name: Build and Test
+    runs-on: ubuntu-latest
+
+    steps:
+      # Check-out repo
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      # Ensure node version is great enough
+      - name: Use Node.js v12.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+
+      # Install dependencies
+      - name: Install rush
+        run: npm install -g @microsoft/rush
+      - name: Install dependencies
+        run: rush install
+
+      # Get datetime
+      - name: Get datetime for alpha release name
+        id: datetime
+        run: |
+          echo "::set-output name=datetime::$(date +'%Y%m%d.%H%M')"
+
+      # Bump alpha package versions
+      - name: Bump alpha package versions
+        run: node common/config/node_modules/beachball/bin/beachball canary --canary-name alpha-${{ steps.datetime.outputs.datetime }} --tag dev --no-publish
```

Comment:
Should this also be `1.0.0-alpha-${{ ... }}`.

Even better, should the step above simply print out the version string to use (so it can be used here as well as in the upload tarball step).